src/FNA3D_Driver_Vulkan.c

```diff
@@ -229,7 +233,7 @@ typedef enum CreateSwapchainResult
 {
 	CREATE_SWAPCHAIN_FAIL,
 	CREATE_SWAPCHAIN_SUCCESS,
-	CREATE_SWAPCHAIN_SURFACE_ZERO,
+	CREATE_SWAPCHAIN_SURFACE_ZERO
```

Comment:
Can probably bring back the comma if only to shrink the diff size

```suggestion
	CREATE_SWAPCHAIN_SURFACE_ZERO,
```